react/ButtonAction/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import styles from './styles.styl'
+import Icon from '../Icon'
+
+const usableByIcon = icon => {
+  const isSvgSymbol = icon && !!icon.id
+  const isIconIdentifier = typeof icon === 'string'
+  return isSvgSymbol || isIconIdentifier
+}
```

Comment:
I think this should be the responsibility of Icon to check fo this and not crash if there is a problem. Otherwise this helper should be used in every possible component that uses `Icon`.

Reply:
I do agree, I took that form the `<Button />` component and it felt redundant.